### DISCUSSION TITLE
Facing 'Error: Command "npm run build" exited with SIGABRT' when deploying Next.js app on Vercel

### DISCUSSION THREAD

**Discussion by pingpongdoctor** [vercel/next.js#60803] [App Router]
The error takes place after I set up nextjs.config.mjs to make plaiceholder work with Next.js:

import withPlaiceholder from '@plaiceholder/next';

/** @type {import('next').NextConfig} */
const nextConfig = {
  images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: 'cdn.sanity.io',
      },
    ],
  },
  experimental: {
    taint: true,
  },
  // ...other config settings
};

export default withPlaiceholder(nextConfig);

Anybody knows why this happen? Please help!

**Comment by swiiny** [comment]
Hey, I recommend trying the following configuration in your `next.config.mjs` file to address the issue with `plaiceholder` and Next.js:

```javascript
import withPlaiceholder from '@plaiceholder/next';

const nextConfig = {
  transpilePackages: ['@plaiceholder/next'],
  images: {
    domains: ['cdn.sanity.io'],
  },
  // ...other config settings
};

export default withPlaiceholder(nextConfig);
```

This setup includes the necessary transpilation for `@plaiceholder/next` and specifies the domains for your images, ensuring they're correctly handled by Next.js. Make sure to include any other required configurations that are specific to your project.

**Comment by EvansRobbie** [comment]
Another work around is to downgrade the sharp package to `sharp@0.32.6`.

**Comment by orlando-guy** [comment]
I faced the same issue; I had to remove `Plaiceholder` from my project to fix it.
